sentry/values.yaml

```diff
@@ -107,54 +98,33 @@ hooks:
         memory: 1Gi
 
 system:
-  url: ''
-  adminEmail: ''
+  url: ""
+  adminEmail: ""
   secretKey: 'icLq77rCyY_qrMMpXa6TQNjkDV6mU!c'
   public: false #  This should only be used if you’re installing Sentry behind your company’s firewall.
 
 mail:
```

Comment:
Should this key stay `email` for interop with the old chart?

Reply:
Thanks for your report. I'll take care of that in the next PR

Reply:
Actually this one is not the previous chart.